docs/contents/user-docs/install.md

```diff
@@ -143,27 +85,20 @@ $ helm list --namespace $ACK_K8S_NAMESPACE -o yaml
   updated: 2020-12-17 13:09:17.309002201 -0500 EST
 ```
 
-## Static Kubernetes manifests
+## Install an ACK service controller with static Kubernetes manifests
 
-If you prefer not to use Helm, you may install a service controller using
-static Kubernetes manifests that are included in the source repository for an
-ACK service controller.
+If you prefer not to use Helm, you may install an ACK service controller using static Kubernetes manifests that are included in the source repository. 
 
-Static Kubernetes manifests that install the individual service controller as a
-Kubernetes `Deployment`, along with the relevant Kubernetes RBAC resources are
-available in the `config/` directory of the associated ACK service controller's
-source repository.
+Static Kubernetes manifests install an individual service controller as a Kubernetes deployment, including the relevant Kubernetes RBAC resources. Static Kubernetes manifests are available in the `config/` directory of the associated ACK service controller's source repository.
```

Comment:
I'd like to also point out there is a namespaced-installation overlay in the `config/overlays/namespaced` directory. But I can draft up a small explanatory paragraph about overlays in the future.